internal/app/app.go

```
@@ -3,64 +3,78 @@ package app
+        health.Module,
+        apiauth.Module,
+        // Server start/stop hooks
+        fx.Invoke(
+            // Manage postgres storage lifecycle
+            postgres.ManageLifecycle,
+            redisStorage.ManageLifecycle,
+            // Manage HTTP server lifecycle
+            func(lc fx.Lifecycle, srv *server.Server, router *gin.Engine) {
+                lc.Append(fx.Hook{
+                    OnStart: func(ctx context.Context) error {
+                        go srv.ListenAndServe(router)
```

golangci-lint (1.64.8)

Error return value of `srv.ListenAndServe` is not checked

(errcheck)